### ISSUE TITLE
macos: bundle.sh can't clone

### ISSUE THREAD

**Issue by smammy** [quodlibet/quodlibet#3393]
Steps to reproduce
------------------
1. Build quodlibet per [macos bundle instructions](https://github.com/quodlibet/quodlibet/tree/master/dev-utils/osx_bundle#creating-a-bundle).
2. Run `./bundle.sh` as instructed.

Expected Output
---------------
```
('Warning, library not available in any prefix:', '/usr/local/opt/gdbm/lib/libgdbm.6.dylib')
('Warning, library not available in any prefix:', '/usr/local/opt/gdbm/lib/libgdbm.6.dylib')
objc[66429]: Class GNotificationCenterDelegate is implemented in both /Users/sam/Files/Projects/Music/quodlibet/dev-utils/osx_bundle/_build/._launcher.app/Contents/Resources/lib/libgio-2.0.0.dylib (0x1046ac480) and /Users/sam/Files/Projects/Music/quodlibet/dev-utils/osx_bundle/_home/jhbuild_prefix/lib/libgio-2.0.0.dylib (0x104935480). One of the two will be used. Which one is undefined.
[thousands of lines, removed]
Listing '/Users/sam/Files/Projects/Music/quodlibet/dev-utils/osx_bundle/_build/QuodLibet.app/Contents/Resources/lib/python3.7/xmlrpc'...
hdiutil: attach: WARNING: ignoring IDME options (obsolete)
hdiutil: attach: WARNING: ignoring IDME options (obsolete)
```

Command exit staus is 0 and a DMG file is produced in `_build`.

Actual Output
-------------
```
('Warning, library not available in any prefix:', '/usr/local/opt/gdbm/lib/libgdbm.6.dylib')
('Warning, library not available in any prefix:', '/usr/local/opt/gdbm/lib/libgdbm.6.dylib')
objc[93697]: Class GNotificationCenterDelegate is implemented in both /Users/sam/Files/Projects/Music/quodlibet/dev-utils/osx_bundle/_build/._launcher.app/Contents/Resources/lib/libgio-2.0.0.dylib (0x108617480) and /Users/sam/Files/Projects/Music/quodlibet/dev-utils/osx_bundle/_home/jhbuild_prefix/lib/libgio-2.0.0.dylib (0x10889b480). One of the two will be used. Which one is undefined.
[thousands of lines, removed]
Compiling '/Users/sam/Files/Projects/Music/quodlibet/dev-utils/osx_bundle/_build/Application.app/Contents/Resources/lib/python3.7/zipapp.py'...
Compiling '/Users/sam/Files/Projects/Music/quodlibet/dev-utils/osx_bundle/_build/Application.app/Contents/Resources/lib/python3.7/zipfile.py'...
fatal: repository '../' does not exist
```
Command exit status is 128 and no DMG file is produced.

Full output here: https://gist.github.com/smammy/cf03841ccb02a42cea4f8e076274e142

Test System
-----------

### Which version of Quod Libet?
Git commit 90fc5d78f8ec

### Which operating system 
macOS 10.15.5

### If it's audio-related, what back-end?
n/a

Additional Information
----------------------
n/a

**Comment by lazka** [comment]
Likely fallout from the dev-utils directory move, so everything is one directory level off. I'll have a look.

**Comment by smammy** [comment]
@lazka I fixed it in PR #3394. Not sure why it isn't showing up here.

**Comment by lazka** [comment]
Thanks
